templates/partGun_GSD_template.py

```diff
@@ -33,48 +33,12 @@
     input = cms.untracked.int32(DUMMYEVTSPERJOB)
 )
 
-process.RandomNumberGeneratorService = cms.Service("RandomNumberGeneratorService",
-                                                   generator = cms.PSet(
-                initialSeed = cms.untracked.uint32(DUMMYSEED),
-                engineName = cms.untracked.string('HepJamesRandom')
-                ),
-                                                   mix = cms.PSet(
-                initialSeed = cms.untracked.uint32(DUMMYSEED),
-                engineName = cms.untracked.string('TRandom3')
-                ),
-                                                   VtxSmeared = cms.PSet(
-                initialSeed = cms.untracked.uint32(DUMMYSEED),
-                engineName = cms.untracked.string('TRandom3')
-                ),
-                                                   g4SimHits = cms.PSet(
-                initialSeed = cms.untracked.uint32(DUMMYSEED),
-                engineName = cms.untracked.string('TRandom3')
-                ),
-                                                   simMuonGEMDigis = cms.PSet(
-                initialSeed = cms.untracked.uint32(DUMMYSEED),
-                engineName = cms.untracked.string('TRandom3')
-                ),
-                                                   simMuonME0Digis = cms.PSet(
-                initialSeed = cms.untracked.uint32(DUMMYSEED),
-                engineName = cms.untracked.string('TRandom3')
-                ),
-                                                   simMuonCSCDigis = cms.PSet(
-                initialSeed = cms.untracked.uint32(DUMMYSEED),
-                engineName = cms.untracked.string('TRandom3')
-                ),
-                                                   simMuonDTDigis = cms.PSet(
-                initialSeed = cms.untracked.uint32(DUMMYSEED),
-                engineName = cms.untracked.string('TRandom3')
-                ),
-                                                   simMuonRPCDigis = cms.PSet(
-                initialSeed = cms.untracked.uint32(DUMMYSEED),
-                engineName = cms.untracked.string('TRandom3')
-                )
-                                                   )
+process.RandomNumberGeneratorService.generator.initialSeed = cms.untracked.uint32(DUMMYSEED)
```

Comment:
Did you check that if you run two jobs, the events are different by using only this line and removing those above?